### PR TITLE
Feat/metrics

### PR DESCRIPTION
**Pull request by logan-anderson**
Adds basic open source metrics see in https://github.com/tinacms/tinacms/issues/2449.

### CODE REVIEW

**Comment by jeffsee55** [comment]
Closes https://github.com/tinacms/tinacms/issues/2449

**Comment by logan-anderson** [comment]
closes #2449

File: packages/@tinacms/cli/src/cmds/audit/index.ts [hunk 1]
```diff
@@ -23,10 +23,19 @@ import { auditCollection, auditDocuments } from './audit'
 import { logger } from '../../logger'
 import chalk from 'chalk'
 import prompts from 'prompts'
+import { Telemetry } from '@tinacms/metrics'
 
 const rootPath = process.cwd()
 
 export const audit = async (ctx: any, next: () => void, options) => {
+  const t = new Telemetry({ disabled: options.noTelemetry })
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/cmds/audit/index.ts:31]: nitpick - a better variable name would be helpful here (and for the other sections like this).

File: packages/@tinacms/cli/src/cmds/audit/index.ts [hunk 2]
```diff
@@ -23,10 +23,19 @@ import { auditCollection, auditDocuments } from './audit'
...
+  await t.submitRecord({
+    event: {
+      name: 'tinacms:audit:invoke',
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/cmds/audit/index.ts:34]: Following the convention from this other ones, maybe this should be `tinacms:cli:audit:invoke`

**Review by jeffsee55 — COMMENTED** [review]
This looks great, a couple of small comments. One thing we may also want to include (though it may be too noisy) would be [here](https://github.com/tinacms/tinacms/blob/132c630b97bcf29fb98a91784a9444a8bad4a9b9/packages/@tinacms/cli/src/cmds/start-server/server.ts#L59). If `result.errors`, we could report those.

File: packages/@tinacms/cli/src/cmds/audit/index.ts (hunk 1 above)
Comment (logan-anderson) [reply packages/@tinacms/cli/src/cmds/audit/index.ts]: Changed!

File: packages/@tinacms/cli/src/cmds/audit/index.ts (hunk 2 above)
Comment (logan-anderson) [reply packages/@tinacms/cli/src/cmds/audit/index.ts]: Definitely

**Comment by logan-anderson** [comment]
@jeffsee55 Updated based on your comments. I also broke this piece of work into its own issue:https://github.com/tinacms/tinacms/issues/2524

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**